persistent-data-provider/schema.prisma

```diff
@@ -0,0 +1,16 @@
+datasource postgresql {
+  url      = env("DATABASE_URL")
+  provider = "postgresql"
+}
+
+generator client {
+  provider = "prisma-client-js"
+  output   = "node_modules/.prisma/client"
+}
+
+model Business {
```

Comment:
I take it that this is generated from the `schema.ts` somehow?

Reply:
Yes